java/src/main/java/ai/rapids/cudf/CSVWriterOptions.java

```diff
@@ -31,6 +34,9 @@ public class CSVWriterOptions {
   private String nullValue = "";
   private String falseValue = "false";
   private String trueValue = "true";
+  // Quote style used for CSV data.
+  // Currently, `true` corresponds to `MINIMAL`, `false` to `NONE`.
```

Comment:
I don't think this comment is correct any more.

Reply:
Thank you. I thought I'd gotten it already.